style-burger.css

```diff
@@ -0,0 +1,86 @@
+h1 {
+  color: violet;
+}
+
+#menu__toggle {
+  opacity: 0;
+}
+
+#menu__toggle:checked ~ .menu__btn > span {
+  transform: rotate(45deg);
+}
+#menu__toggle:checked ~ .menu__btn > span::before {
+  top: 0;
+  transform: rotate(0);
+}
+#menu__toggle:checked ~ .menu__btn > span::after {
+  top: 0;
+  transform: rotate(90deg);
+}
+#menu__toggle:checked ~ .menu__box {
+  visibility: visible;
+  left: 0;
+}
+
+.menu__btn {
+  display: flex;
+  align-items: center;
+  position: fixed;
+  top: 20px;
+  left: 20px;
+  width: 26px;
+  height: 26px;
+  cursor: pointer;
+  z-index: 1;
+}
+
+.menu__btn > span,
+.menu__btn > span::before,
```

Comment:
Codacy found an issue: [Expected selector ".menu__btn > span::before" to come before selector "#menu__toggle:checked ~ .menu__btn > span::before" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/burger-css17/pullRequest?prid=6077574)